platform/platform-impl/src/com/intellij/openapi/wm/impl/status/TextPanel.java

```diff
@@ -107,15 +107,7 @@ private static String splitText(final JLabel label, final String text, final int
 
     final String[] lines = UIUtil.splitText(text, fontMetrics, widthLimit, ' ');
 
-    final StringBuilder result = new StringBuilder();
-    for (int i = 0; i < lines.length; i++) {
-      final String line = lines[i];
-      if (i > 0) {
-        result.append('\n');
-      }
-      result.append(line);
-    }
-    return result.toString();
+    return String.join("\n", lines);
```

Comment:
Note that this private method was completely unused for a long time. I removed it in the subsequent commit.

Reply:
Right, it was highlighted as unused, however I thought it's too risky ro remove it